src/code/models/codap-connect.ts

```diff
@@ -91,75 +91,78 @@ export class CodapConnect {
+            this.standaloneMode = true;
+            this.graphStore.setCodapStandaloneMode(true);
+          }
 
-        // We check for game state in either the frame (CODAP API 2.0) or the dataContext
-        // (API 1.0). We ignore the dataContext if we find game state in the interactiveFrame
-        const state = (frame != null ? frame.values.savedState : undefined) ||
-                __guard__(__guard__(context != null ? context.values : undefined, x1 => x1.contextStorage), x => x.gameState);
+          // We check for game state in either the frame (CODAP API 2.0) or the dataContext
+          // (API 1.0). We ignore the dataContext if we find game state in the interactiveFrame
+          const state = (frame != null ? frame.values.savedState : undefined) ||
+                  __guard__(__guard__(context != null ? context.values : undefined, x1 => x1.contextStorage), x => x.gameState);
 
-        if (state != null) {
-          this.graphStore.deleteAll();
-          this.graphStore.loadData(state);
-          return this._initialSyncAttributeProperties(null, true);
+          if (state != null) {
+            this.graphStore.deleteAll();
+            this.graphStore.loadData(state);
+            return this._initialSyncAttributeProperties(null, true);
+          }
+        } else {
+          return log.info("null response in codap-connect codapPhone.call");
         }
-      } else {
-        return log.info("null response in codap-connect codapPhone.call");
-      }
-    });
+      });
 
 
-    // check if we already have a datacontext (if we're opening a saved model).
-    // if we don't create one with our collections. Then kick off init
-    this.codapPhone.call({
-      action: "get",
-      resource: `dataContext[${this.dataContextName}]`
-    }
-    , ret => {
-      // ret==null is indication of timeout, not an indication that the data set
-      // doesn't exist.
-      if (!ret || ret.success) {
-        let attrs;
-        if (attrs = (__guard__(__guard__(__guard__(ret != null ? ret.values : undefined, x2 => x2.collections), x1 => x1[1]), x => x.attrs) != null)) {
-          this._initialSyncAttributeProperties(attrs);
-        }
-      } else {
-        this._createDataContext();
+      // check if we already have a datacontext (if we're opening a saved model).
+      // if we don't create one with our collections. Then kick off init
+      this.codapPhone.call({
+        action: "get",
+        resource: `dataContext[${this.dataContextName}]`
       }
-      this.updateExperimentColumn();
-      return this._getExperimentNumber();
+      , ret => {
+        // ret==null is indication of timeout, not an indication that the data set
+        // doesn't exist.
+        if (!ret || ret.success) {
+          let attrs;
+          if (attrs = (__guard__(__guard__(__guard__(ret != null ? ret.values : undefined, x2 => x2.collections), x1 => x1[1]), x => x.attrs) != null)) {
```

Comment:
I realize this isn't new code but this is ... something. 

Assignment in a conditional evaluation, assigned to a tertiary with function calls such as `(__guard__(__guard__(__guard__(`   

Not a fan.

Reply:
This is all cruft over from the CoffeScript -> TypeScript conversion.